locales/en/plugin__dfr-console.json

```diff
@@ -55,22 +55,43 @@
   "Access Key Field": "Access Key Field",
   "Switch to Secret": "Switch to Secret",
   "Secret Key Field": "Secret Key Field",
-  "Create {{kind}}": "Create {{kind}}",
+  "Buckets": "Buckets",
+  "Resource Page": "Resource Page",
+  "Filter by policy name": "Filter by policy name",
+  "Filter by policy label": "Filter by policy label",
+  "Create New": "Create New",
```

Comment:
```suggestion
  "Create Data resource": "Create Data resource",
```

Reply:
In UX it is mentioned as Create New 

https://marvelapp.com/prototype/2e221bb6/screen/86409476

Reply:
Done